services/jsdelivr/jsdelivr-hits-github.service.js

```diff
@@ -6,17 +7,27 @@ export default class JsDelivrHitsGitHub extends BaseJsDelivrService {
     pattern: ':period(hd|hw|hm|hy)/:user/:repo',
```

Comment:
TODO: enum

Reply:
_sigh_ and also work out why the liberapay tests are failing (they pass locally)

Reply:
I've just reverted this for now